operators/community-windows-machine-config-operator/5.1.0/manifests/community-windows-machine-config-operator.5.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,329 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-06-01T09:47:21Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.0.0'
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.openshift.io/valid-subscription: '["Red Hat OpenShift support for Windows Containers"]'
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: Red Hat
+  name: community-windows-machine-config-operator-5.1.0
+  namespace: placeholder
```

Comment:
s/placeholder/openshift-windows-machine-config-operator

Reply:
we will actually need 'placeholder'. Doc ref: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#csv-metadata